comid/tdx-profile/example_pce_refval_test.go

```diff
@@ -145,13 +161,14 @@ func decodePCEMValExtensions(m *comid.Measurement) error {
 		return fmt.Errorf("failed to decode teetcbcompsvn from measurement extensions")
```

Comment:
Please use `errors.New`

This comment also applies to all the many other places where there is no need for formatted printing.

Reply:
Done, thanks for noticing this over-sight!